frontend/stylesheets/screens/_home.scss

```
@@ -28,14 +28,14 @@ body.home .app-content {
 }
 
 body.home .list-speaker {
-  margin-top: 2rem;
+margin-top: 2rem;
 }
 
 body.home .list-sponsor {
   padding-left: 0;
 
   @include media-breakpoint-up('sm') {
     width: 65%;
-    margin: 0 auto;
+    margin: 0 auto 24px;
```

Sizing must be in rem as explained above:

```suggestion
    margin: 0 auto 1.5rem;
```